ocp_install_env.sh

```diff
@@ -13,7 +13,7 @@ export EXTERNAL_SUBNET="192.168.111.0/24"
 # The release we default to here is pinned and known to work with our current
 # version of kni-installer.
 #
-export OPENSHIFT_RELEASE_IMAGE="registry.svc.ci.openshift.org/kni/release:4.1.0-rc.5-kni.1"
+export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE="registry.svc.ci.openshift.org/kni/release:4.1.0-rc.5-kni.1"
```

Comment:
This would mean that we'd always be overriding the release payload pullspec in the binary and not testing that path, even where we're not building the installer from git

How about in the git case, we do:

```
$ RELEASE_IMAGE="registry.svc.ci.openshift.org/kni/release:4.1.0-rc.5-kni.1" TAGS="libvirt ironic" hack/build.sh
```

so that we get the pullspec built into the binary in that case too